sdk/appconfiguration/azappconfiguration/client_api_revisions.go

```diff
@@ -0,0 +1,56 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azappconfiguration
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/appconfiguration/azappconfiguration/internal/generated"
+)
+
+// GetRevisionsPager is a Pager for revision list operations.
+//
+// NextPage should be called first. It fetches the next available page of results from the service.
+// If the fetched page contains results, the return value is true, else false.
+// Results fetched from the service can be evaluated by calling PageResponse on this Pager.
+// If the result is false, the value of Err() will indicate if an error occurred.
+type GetRevisionsPager interface {
```

Comment:
Should be ListRevisionsPager, also the new guidance is to return a struct not an interface (it might not be updated in the guidelines).